qsimcirq/qsim_circuit.py

```diff
@@ -305,7 +305,7 @@ def to_matrix(op: cirq.ops.GateOperation):
         cirq.decompose(op, fallback_decomposer=to_matrix, keep=has_qsim_kind)
         for op in moment
       ]
-      moment_length = max(len(gate_ops) for gate_ops in ops_by_gate)
+      moment_length = max([0] + [len(gate_ops) for gate_ops in ops_by_gate])
```

Comment:
Alternatively, can pass a `default` argument to `max`, which avoids having to create an explicit list:
```suggestion
      moment_length = max((len(gate_ops) for gate_ops in ops_by_gate), default=0)
```

Reply:
Done.